yogstation/code/modules/mob/living/carbon/human/species_types/mothmen.dm

```diff
@@ -0,0 +1,26 @@
+/datum/species/moth
+	var/MothLightRange = 15 //In what radius shall moths look for lights to fly at?
+	var/MothLightProb = 2 //Don't want them doing it tooooo often..Unless an admin changes it :^) So change this back to 5 after testing
```

Comment:
this is called what, every 2 seconds, so with a probability of 2 this'll happen once every 2 minutes. I'm not necessarily saying that's too often, just saying that's how it is.